src/components/Edges/wrapEdge.tsx

```diff
@@ -146,14 +149,7 @@ export default (EdgeComponent: ComponentType<EdgeProps>) => {
             onMouseEnter={onEdgeUpdaterMouseEnter}
             onMouseOut={onEdgeUpdaterMouseOut}
           >
-            <circle
-              className="react-flow__edgeupdater"
-              cx={sourceX}
-              cy={sourceY}
-              r={10}
-              stroke="transparent"
-              fill="transparent"
-            />
+            <EdgeAnchorStart {...props} />
```

Comment:
We only do prop spreading when all props are used by the component.

Reply:
fixed